src/modules/launcher/Plugins/Community.PowerToys.Run.Plugin.VSCodeWorkspaces/Community.PowerToys.Run.Plugin.VSCodeWorkspaces.csproj

```diff
@@ -11,20 +11,17 @@
 		<useWPF>true</useWPF>
 		<AppendTargetFrameworkToOutputPath>false</AppendTargetFrameworkToOutputPath>
 		<AppendRuntimeIdentifierToOutputPath>false</AppendRuntimeIdentifierToOutputPath>
-		<Platforms>x64</Platforms>
-    <PlatformTarget>x64</PlatformTarget>
+		<OutputPath>..\..\..\..\$(Platform)\$(Configuration)\modules\launcher\Plugins\VSCodeWorkspaces\</OutputPath>
```

Comment:
Comparing the diff, I think your missing a `..` here too. I assume on other plugins this will be the same.